testing/test_cacheprovider.py

```diff
@@ -925,3 +925,22 @@ def test_does_not_create_boilerplate_in_existing_dirs(testdir):
     assert os.path.isdir("v")  # cache contents
     assert not os.path.exists(".gitignore")
     assert not os.path.exists("README.md")
+
+
```

Comment:
i propose importing that value instead of dupicating

Reply:
fixed